src/ripple/peerfinder/impl/Logic.h

```diff
@@ -1088,8 +1088,8 @@ class Logic
                 item ["inbound"]    = "yes";
             if (slot.fixed())
                 item ["fixed"]      = "yes";
-            if (slot.cluster())
-                item ["cluster"]    = "yes";
+            if (slot.reserved())
+                item ["reserved"]   = "yes";
```

Comment:
Similar to Counts.h, I don't _think_ this is written to a public API, so I'm guessing the rename from `cluster` to `reserved` won't hurt anybody. But that would be a good thing to research and confirm.  And maybe leave a comment once you've done the research.  Thanks.